eZ/Bundle/EzPublishCoreBundle/Composer/ScriptHandler.php

```diff
@@ -25,12 +25,13 @@ public static function dumpAssets(CommandEvent $event)
         $options = self::getOptions($event);
         $appDir = $options['symfony-app-dir'];
         $webDir = $options['symfony-web-dir'];
-        $env = isset($options['ezpublish-asset-dump-env']) ? $options['ezpublish-asset-dump-env'] : '';
+        // @deprecated: Use of ezpublish-asset-dump-env is deprecated, use standard SYMFONY_ENV instead.
+        $env = !empty($options['ezpublish-asset-dump-env']) ? $options['ezpublish-asset-dump-env'] : (getenv('SYMFONY_ENV') ?: '');
 
         if (!$env) {
             $env = $event->getIO()->ask(
-                "<question>Which environment would you like to dump production assets for?</question> (Default: 'prod', type 'none' to skip) ",
-                'prod'
+                "<question>'SYMFONY_ENV' environment variable not defined, which environment would you like to dump production assets for?</question> (Default: 'dev', type 'none' to skip) ",
+                'dev'
```

Comment:
@bdunogier about automated scripts: it was hard to automate before this PR, as you normally have only 1 composer.json in the repo but use it in many envs, so you can not put in it the name of one environment. The only way for me to actually automate it is to have shell code alter the composer.json after a git pull and before a composer-install call.
In other words: someone might have relied on the old default, but I think that it is not many users / not the heavy-automating users